setup.py

```diff
@@ -49,25 +58,25 @@
 
 
 setup(
-    name="<PYPI_NAME>",
+    name="docs-build",
     use_scm_version=True,
     setup_requires=["setuptools_scm"],
-    description="""<PYPI_NAME>: <SHORT_DESCRIPTION>""",
+    description="""docs-build: Build Sphinx Documentation for ApeWorX plugins""",
```

Comment:
I am not sure why we are using triple quotes here

```suggestion
    description="docs-build: Build Sphinx Documentation for ApeWorX plugins",
```